frontend/app/src/components/form/object-form.tsx

```diff
@@ -115,13 +115,13 @@ const GenericSelector = (props: GenericSelectorProps) => {
   );
 };
 
-const NodeWithProfileForm = ({ kind, currentProfile, ...props }: ObjectFormProps) => {
+const NodeWithProfileForm = ({ kind, currentProfiles, ...props }: ObjectFormProps) => {
   const nodes = useAtomValue(schemaState);
   const generics = useAtomValue(genericsState);
   const profiles = useAtomValue(profilesAtom);
-  const [profileSelected, setProfileSelected] = useState<
-    Record<string, Pick<AttributeType, "value" | "__typename">> | undefined
-  >(currentProfile);
+
+  const [selectedProfiles, setSelectedProfiles] = useState<IProfileSchema[] | undefined>();
```

Comment:
because the selectedProfiles need to have all the data (profiles object with the corresponding attributes and their values)

all that is fetched by the profile selector, if there is any current profile (for which we only have the ids and names) it will aggregate the profiles data (attributes and values) and set them as default